CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -44,6 +46,11 @@ class EventDetailsVC: UIViewController {
                 }
             }
         }
+        
+        // gray out the facebook button if there is no link
```

Comment:
Doesn't this code completely remove it? The comment seems inaccurate

Reply:
Also, seems like you should be checking the null case as well as checking the empty string case